Makefile

```diff
@@ -0,0 +1,5 @@
+lint:
+	pre-commit run
```

Comment:
What is this? Am I meant to have a `pre-commit` binary?

Reply:
Indeed, linting is done via pre-commit which is configured [here](https://github.com/utilitywarehouse/kafka-cluster-config/blob/bbe2873818d93da5929e2835251e25017cabd48d/.pre-commit-config.yaml). I moved the documentation here: https://github.com/utilitywarehouse/kafka-cluster-config/blob/bbe2873818d93da5929e2835251e25017cabd48d/README.md#linting